base/src/main/java/com/smartdevicelink/managers/screen/menu/BaseVoiceCommandManager.java

```diff
@@ -194,6 +202,24 @@ private void updateIdsOnVoiceCommands(List<VoiceCommand> voiceCommands) {
         }
     }
 
+    List<VoiceCommand> removeEmptyVoiceCommands(List<VoiceCommand> voiceCommands) {
+        List<VoiceCommand> validatedVoiceCommands = new ArrayList<>();
+        for (VoiceCommand voiceCommand : voiceCommands) {
+            List<String> voiceCommandStrings = new ArrayList<>();
```

Comment:
I changed it to removing null voiceCommands in `isVoiceCommandsUnique` because it would hit the NPE there first.

Reply:
I suggest:
- Keeping the null check in both methods just in case in future we change the method call order
- Do not remove the null items from VoiceCommands and just skip them instead because we are not supposed to modify the list that the div passes